src/MgvOfferTaking.sol

```diff
@@ -306,12 +306,12 @@ abstract contract MgvOfferTaking is MgvHasOffers {
             mor.leaf = mor.leaf.setBinFirst(bin, sor.offerId);
           }
 
-          // no need to test whether level0 has been reached since by default its stored in local
+          // no need to test whether level1 has been reached since by default its stored in local
```

Comment:
```suggestion
          // no need to test whether level1 has been reached since by default it's stored in local
```

Reply:
fixed in b629e86b9d32fcd905bf8643ef65408841751c08